src/pages/docs/octopus-rest-api/cli/octopus-account-aws-create.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 layout: src/layouts/Default.astro
 pubDate: 2023-01-01
-modDate: 2024-02-01
+modDate: 2024-09-12
```

Comment:
We need to be careful about signalling a modification when the body of the page hasn't changed. Google will interpret modified date changes with no change to content as spam. This could impact them accepting modified dates for our whole site as they treat this as a binary "trust/don't trust" decision.